content/en/getting-started/installation.md

```diff
@@ -24,7 +24,28 @@ All of these tools must be in your PATH, so that kluctl can easily invoke them.
    ```bash
    # To download a specific version, replace the kluctl_version with the version you need for example kluctl_version={{< param "fullversion" >}}
    kluctl_version=$(curl --silent -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kluctl/kluctl/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
-   curl -LO "https://github.com/kluctl/kluctl/releases/download/${kluctl_version}/kluctl-linux-amd64"
+   curl -LO "https://github.com/kluctl/kluctl/releases/download/${kluctl_version}/kluctl_${kluctl_version:1}_Linux_x86_64.tar.gz"
```

Comment:
Maybe we should add a note somewhere that this is an example for linux amd64 or even better try to make this switchable for each os/arch. For example like golang does it https://go.dev/doc/install :innocent: .